themes/hugo-rladies/layouts/events/single.html

```diff
@@ -1,335 +1,164 @@
-                <i class="calendar-icon tui-full-calendar-dropdown-arrow"></i>
-            </button>
-            <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu-calendarType">
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-daily">
-                        <i class="calendar-icon ic_view_day"></i>Daily
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weekly">
-                        <i class="calendar-icon ic_view_week"></i>Weekly
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-monthly">
-                        <i class="calendar-icon ic_view_month"></i>Month
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weeks2">
-                        <i class="calendar-icon ic_view_week"></i>2 weeks
-                    </a>
-                </li>
-                <li role="presentation">
-                    <a class="dropdown-menu-title" role="menuitem" data-action="toggle-weeks3">
-                        <i class="calendar-icon ic_view_week"></i>3 weeks
-                    </a>
-                </li>
-            </ul>
-        </span>
-        <span id="menu-navi">
-            <button type="button" class="btn btn-primary btn-sm move-today" data-action="move-today">Today</button>
-            <button type="button" class="btn btn-primary btn-sm move-day" data-action="move-prev">
-                <i class="fa fa-chevron-circle-left ic-arrow-line-left" data-action="move-prev"></i>
-            </button>
-            <button type="button" class="btn btn-primary btn-sm move-day" data-action="move-next">
-                <i class="fa fa-chevron-circle-right ic-arrow-line-right" data-action="move-next"></i>
-            </button>
-        </span>
-        <span id="renderRange" class="render-range"></span>
+  </center>
+  <div id="top">
+    <div style="float: left">
+      Timezone:
+      <select id="time-zone-selector">
+        <option value="America/Los_Angeles">
+          Los Angeles, Pacific Daylight Time (PDT)
+        </option>
+        <option value="utc">Universal Time Coordinated (UTC)</option>
+        <option value="Africa/Lagos">Lagos, West Africa Time (WAT)</option>
+        <option value="Europe/Zurich">
+          Zurich, Central European Summer Time (CEST)
+        </option>
+        <option value="Asia/Kolkata">Mumbai, India Standard Time (IST)</option>
+        <option value="Australia/Perth">
+          Perth, Australian West Standard Time (AWST)
+        </option>
+      </select>
+    </div>
+    <div style="float: right">
```

Comment:
```suggestion
    <div class="float-right">
```

change to bootstrap class